#include <CPPExtensions/system.hpp>
#include <CPPExtensions/Exception.hpp>
using namespace CPPExtensions;

int main() {
	Utils::print(Utils::std_out, "hey there\n");
	Utils::RunError("test");
}
